kiosk/src/Components/PlayingGame.tsx

```diff
@@ -16,6 +16,12 @@ export default function PlayingGame() {
         return getLaunchedGame();
     }, [kiosk.launchedGameId]);
 
+    const makeRunUrl = () => {
+        return pxt.webConfig?.runUrl
+        ? "https://arcade.makecode.com" + pxt.webConfig.runUrl
```

Comment:
this will make it so staging still points at prod sim, which isn't ideal -- if leaving it as just a relative url (that is, just using .runUrl with no host domain) works that'd be great, otherwise `window.location.origin` instead of `https://arcade.makecode.com` would be good as well

Reply:
So, I've been testing it with just using `pxt.webConfig.runUrl`, and it looks like it works. I'm not sure I understand why, though. `pxt.Webconfig.runUrl` only has the end path of the url, not the domain. Does the browser just resolve and say "just use the same root/domain if one isn't explicitly supplied"?

Reply:
(discussed in standup but basically answer is 'yes' / paths get treated as relative to origin)